### PR TITLE
[OSS in Business] Created Contribution.md

### PR DESCRIPTION
**Pull request by harmonyelendu**
To further facilitate participation and growth in OSS whitepaper the contribution.md file points contributors to the right direction by serving as a detailed guide in other to start contributing.

### CODE REVIEW

**Comment by anajsana** [comment]
Thanks for starting this @harmonyelendu !

I assume you still need populate with content the Markdown created in this PR before is ready to be reviewed on my end. Please let me know once is finished 🙂

**Review by harmonyelendu — COMMENTED** [review]
@anajsana  added more context on my proposal for a contributor.md file for OSS in Business White Paper.

File: whitepapers/business-value/Contribution.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Steps to Contribute to OSS Whitepaper! 
+This is basically an Overview of what this document should entail.
```
Comment (anajsana) [inline whitepapers/business-value/Contribution.md:2]:
```suggestion
This document provides an overview of the different ways individuals can contribute as co-creators of the whitepaper, as well as the different roles they can take on.
```

File: whitepapers/business-value/Contribution.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Steps to Contribute to OSS Whitepaper! 
+This is basically an Overview of what this document should entail. 
+
+The Contribution.md file will serve as a guide, providing the needed step by step process of how contributors will collaboratively work on the OSS in business whitepaper, ensuring collective approval from everyone invloved. However given the large scope and extensive work required, I would recommend organising contributors into three groups: They include:
```
Comment (anajsana) [inline whitepapers/business-value/Contribution.md:4]:
```suggestion
The Contribution.md file provides a step by step process of how contributors will work on the OSS in business whitepaper, ensuring collective approval from everyone invloved. Contributions can be organized into three groups:
```

**Review by anajsana — CHANGES_REQUESTED** [review]
Thanks again for drafting the initial doc! I've made some suggestions to changes to the intro 🙂

**Comment by harmonyelendu** [comment]
@anajsana Thanks so much for spotting better words

File: whitepapers/business-value/Contribution.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Steps to Contribute to OSS Whitepaper! 
+This document provides an overview of the different ways individuals can contribute as co-creators of the whitepaper, as well as the different roles they can take on.
+
+The Contribution.md file provides a step by step process of how contributors will work on the OSS in business whitepaper, ensuring collective approval from everyone invloved. Contributions can be organized into three groups:
+
+- Group members: Contributors will be organised into groups based off interest and expertise aligned with the topics and section provided through the outline in the whitepaper. Each group will be led by a leader providing the required structure and ensuring accountability for progress.
+- Group Lead: A group lead will be responsible for cordinating efforts and facilitating the progress of it's memebers. Group leads will serve beyond just guidance for the contributors as they work on their respective sections before syncing through mini presentations during our calls. However, individuals interested in becoming group leads will have to meet set up basic requirements made by the working group before expressing their interest.
```
Comment (anajsana) [inline whitepapers/business-value/Contribution.md:7]:
```suggestion
- Group Chair: A group chair will be responsible for coordinating efforts and facilitating the progress of the working group participants of each section. Group chairs will serve beyond just guidance for the contributors as they work on their respective sections before syncing through mini presentations during Touchpoint calls. However, individuals interested in becoming group chairs will have to meet set up basic requirements made by the working group before expressing their interest.  
```

File: whitepapers/business-value/Contribution.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Steps to Contribute to OSS Whitepaper! 
+This document provides an overview of the different ways individuals can contribute as co-creators of the whitepaper, as well as the different roles they can take on.
+
+The Contribution.md file provides a step by step process of how contributors will work on the OSS in business whitepaper, ensuring collective approval from everyone invloved. Contributions can be organized into three groups:
+
+- Group members: Contributors will be organised into groups based off interest and expertise aligned with the topics and section provided through the outline in the whitepaper. Each group will be led by a leader providing the required structure and ensuring accountability for progress.
```
Comment (anajsana) [inline whitepapers/business-value/Contribution.md:6]:
```suggestion
- Group participants: Contributors will be organised into groups based on interest and expertise aligned with the topics and section provided through the outline in the whitepaper. Each group will be led by a leader providing the required structure and ensuring accountability for progress.
```

File: whitepapers/business-value/Contribution.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+- Group Lead: A group lead will be responsible for cordinating efforts and facilitating the progress of it's memebers. Group leads will serve beyond just guidance for the contributors as they work on their respective sections before syncing through mini presentations during our calls. However, individuals interested in becoming group leads will have to meet set up basic requirements made by the working group before expressing their interest.  
+- The reviewer: This is a group of people that will focus more on reviewing the content of other contributors. Their role will ensure that all the work are well documented in google docs used for compling the contriubutions of all groups forming the whitepaper, while ensuring that the outlines and objectives of the working group for the whitepaper are maintained properly --- Additionally, I would also recommend that the working group should make the google doc used for complilations public so viewers can also serve as reviewers through comments and suggestions which will be addressed in the meeting if they cannot be resolved within google docs.
```
Comment (anajsana) [inline whitepapers/business-value/Contribution.md:8]:
```suggestion
- Reviewer: This is a group of people that will focus more on reviewing the content of other contributors. Their role will ensure that all the work are well documented in google docs used for compling the contributions of all groups forming the whitepaper, while ensuring that the outlines and objectives of the working group for the whitepaper are maintained properly. The working group should make the google doc used for compilations public so viewers can also serve as reviewers through comments and suggestions which will be addressed in the meeting if they cannot be resolved within google docs. Later, the content should be moved into markdown format and added into this repo via Pull Request.
```

File: whitepapers/business-value/Contribution.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+
+Please note: I will be open for any questions and clarity if needed.
```
Comment (anajsana) [inline whitepapers/business-value/Contribution.md:10]:
```suggestion
> Please note: I will be open for any questions and clarity if needed. 
```

**Review by anajsana — CHANGES_REQUESTED** [review]
Thank you! requested a few changes. If you are ok with them, once merged into the branch, is good to go and this PR can be merged on my end

**Review by anajsana — APPROVED** [review]

**Pull request merged**